/*
The concept of Exception is used when trying to handle an unexpected behavior of the program and display it to the end user, whilest also enabling the program to continue operation or different operations for the end user instead of the programing come crashing. 

They are used at the calling program from where the expected error to occur. 

using a try catch block. With try having the program block while catch block containing the handling of the errors
*/
